packages/expo-modules-core/android/src/main/java/expo/modules/core/logging/LogHandler.kt

```diff
@@ -0,0 +1,19 @@
+package expo.modules.core.logging
+
+/**
+ * The interface that needs to be implemented by log handlers.
+ */
+interface LogHandler {
+  /**
+   * Category is always passed in from Logger
+   */
+  val category: String
+
+  /**
+   * For handlers that require additional parameters
+   * (e.g. Android context can be passed in here)
+   */
+  val additionalInfo: Any?
+
```

Comment:
remove (these aren't called externally and interfaces generally don't dictate concrete implementor's properties)

Reply:
These are in fact used internally in the handler classes.

Reply:
Yep, but an interface should only dictate what is needed to be known/used externally.

Reply:
I did a bit more research on this, and find that what I really need to do to match what @tsapeta did in Swift is to use an abstract class, so I'll move to that.